dockstore-client/src/main/java/io/dockstore/client/cli/SwaggerUtility.java

```diff
@@ -43,19 +44,24 @@ private SwaggerUtility() {
                 new String[] { "BEARER" }, type);
     }
 
-    public static void unzipFile(File zipFile, File unzipDirectory) throws IOException {
+    public static void unzipFile(File zipFile, File unzipDirectory, boolean deleteZip) throws IOException {
```

Comment:
I see that deleteZip is most often false. 
In these cases, I normally provide an overloaded method
`unzipFile(File zipFile, File unzipDirectory)` which just calls the other with deleteZip set to false. 
I find this a bit less error prone and causes fewer changes